be/src/runtime/raw_value.h

```diff
@@ -406,6 +420,10 @@ inline uint32_t RawValue::zlib_crc32(const void* v, const TypeDescriptor& type,
         seed = HashUtil::zlib_crc_hash(&int_val, sizeof(int_val), seed);
         return HashUtil::zlib_crc_hash(&frac_val, sizeof(frac_val), seed);
     }
+
+    case TYPE_DECIMAL_V2: {
```

Comment:
you should keep this same with TYPE_DECIMAL, because we use this to distribute data